src/ocm/components/clusterConfiguration/CustomManifestCheckbox.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { Alert, FormGroup } from '@patternfly/react-core';
+import { useField } from 'formik';
+import { getFieldId, HelperText, PopoverIcon } from '../../../common';
+import { OcmCheckbox } from '../ui/OcmFormFields';
+import { useClusterWizardContext } from '../clusterWizard/ClusterWizardContext';
+import DeleteCustomManifestModal from './manifestsConfiguration/DeleteCustomManifestModal';
+import useClusterCustomManifests from '../../hooks/useClusterCustomManifests';
+
+const Label = () => {
+  return (
+    <>
+      Include custom manifests{' '}
+      <PopoverIcon
+        noVerticalAlign
+        bodyContent={
+          <p>
+            Include custom manifests to incorporate third party manifests that are not supported in
+            assisted installer wizard and APIs.
+          </p>
+        }
+      />
+    </>
+  );
+};
+
+type CustomManifestCheckboxProps = { clusterId: string };
+
+const CustomManifestCheckbox = ({ clusterId }: CustomManifestCheckboxProps) => {
```

Comment:
Please take a look at this, something about this behaviour doesn't seem quite right...


https://user-images.githubusercontent.com/87187179/229773899-16f84ac7-b7e4-407f-8fe4-66412e0ffd10.mp4

Reply:
We need a combination of these two behaviors:
- Create manifests right after you create the cluster when the users click Next (new clusters)
- Create/delete manifests when the users interact with the checkbox/confirm modal on a created cluster (existing clusters)

Reply:
I've opened a bug for the rest of the fields on the Cluster details page. I don't see why this page should work differently than the rest of the app -- everywhere else we have autosave, we don't wait for the users to press the next button.

Reply:
Thanks @jgyselov . I agree with you about these changes.